hyperspace/core/src/command.rs

```diff
@@ -63,6 +67,9 @@ pub struct Cmd {
 	/// Channel version
 	#[clap(long)]
 	version: Option<String>,
+	/// Directory to store the updated configurations
+	#[clap(long)]
+	pub config_output_dir: String,
```

Comment:
We might not need this, we should just override the initial config that was passed